File: A-Lab 2 APPOO/APPOO2_Max/src/Masini.java

public class Masini {
    private String brand;
    private String model;
    private double price;

    public Masini(String brand, String model, double price){
        this.brand = brand;
        this.model = model;
        this.price = price;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }

    public String getBrand(){
        return this.brand;
    }

    public void setModel(String model){
        this.model = model;
    }

    public String getModel(){
        return this.model;
    }

    public void setPrice(double price){
        this.price = price;
    }

    public double getPrice(){
        return this.price;
    }

    @Override
    public String toString(){
        return "["+this.brand+"|"+this.model+"|"+this.price+"$]";
    }
}

File: A-Lab 3 APPOO/Multiset_3/Multiset_3/Masini.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multiset_3
{
    class Masini
    {
        private string brand;
        private string model;
        private double price;

        public Masini(string brand, string model, double price)
        {
            this.brand = brand;
            this.model = model;
            this.price = price;
        }

        public string GetBrand()
        {
            return brand;
        }

        public void SetBrand(string brand)
        {
            this.brand = brand;
        }

        public string GetModel()
        {
            return model;
        }

        public void SetModel(string model)
        {
            this.model = model;
        }

        public double GetPrice()
        {
            return price;
        }

        public void SetPrice(double price)
        {
            this.price = price;
        }

        public override string ToString()
        {
            return "[" + brand + "|" + model + "|" + price + "$]";
        }
    }
}
